src/containers/oracle.js

```diff
@@ -158,70 +155,235 @@ class OraclePage extends React.Component {
+              },
+              ],
+            },
+            messages: [
+              {
+                text: `Result setter ${oracle.resultSetterAddress}`,
+                type: 'default',
+              },
+              {
+                text: `Consensus Threshold ${oracle.consensusThreshold || 0}. This value indicates the amount of BOT needed for result setting.`,
+                type: 'default',
+              },
+
+              {
+                text: 'Upon clicking Confirm, you will need to wait for BOT token to get approved. Those BOT amount will automatically be used to set result afterwards.',
+                type: 'default',
+              },
+            ],
+          },
+          cardAction: {
+            skipToggle: false,
+            beforeToggle: {
+              btnText: 'Set Result',
+              // btnDisabled: (oracle.resultSetterAddress !== selectedWalletAddress),
+            },
+            afterToggle: {
+              showAmountInput: false,
+              btnText: 'Confirm',
+            },
+          },
+        };
+
+        // Add a message to CardInfo to warn that current block has passed set end block
+        if (blockCount > oracle.resultSetEndBlock) {
+          config.cardInfo.messages.push({
+            text: 'Current block number has passed result set end block.',
+            type: 'warn',
+          });
         }
 
-        console.log('oracle', oracle);
-        console.log(`configName: ${configName}`);
+        // Add a message to CardInfo to warn that user is not result setter of current oracle
+        if (oracle.resultSetterAddress !== selectedWalletAddress) {
+          config.cardInfo.messages.push({
+            text: 'You are not the result setter for this topic and cannot set result.',
+            type: 'warn',
+          });
+        }
+      } else if (status === 'VOTING' && token === BOT) {
+        const relatedOracles = _.filter(getOraclesSuccess, (item) => item.topicAddress === oracle.topicAddress);
+        const centralizedOracle = _.find(relatedOracles, (item) => item.token === QTUM);
+
+        config = {
+          name: 'VOTING',
+          breadcrumbLabel: 'Votting',
+          cardInfo: {
+            steps: {
+              current: 3,
+              value: [{
+                title: 'Topic created',
+                description: `Block No. ${(centralizedOracle && centralizedOracle.blockNum) || ''}`,
+              },
+              {
+                title: 'Betting',
+                description: `Block No. ${(centralizedOracle && centralizedOracle.blockNum + 1) || ''} - ${(centralizedOracle && centralizedOracle.endBlock) || ''}`,
+              },
+              {
+                title: 'Result Setting',
+                description: `Block No. ${(centralizedOracle && centralizedOracle.endBlock + 1) || ''} - ${(centralizedOracle && centralizedOracle.resultSetEndBlock)}`,
```

Comment:
do we need to show the `Betting` and `Result Setting` info for a DOracle? Seems like extraneous information to me.

Reply:
For current version we don't separate betting and voting. All four phases are in the same flow so I think showing betting and result settings matches what shows on dashboard tab. Open for discussion.